src/grid/Grid.js

```diff
@@ -15,35 +13,38 @@ class Grid extends Component {
     activeOpacity: 1,
   }
 
+  styles = StyleSheet.create({
```

Comment:
@Monte9 The reason to have styles inside the `Grid` object is because we need access to the `isRow` function. If you know of a better way please let me know